src/webgpu/api/operation/copyBetweenLinearDataAndTexture.spec.ts

```diff
@@ -0,0 +1,742 @@
+export const description = `writeTexture + copyBufferToTexture + copyTextureToBuffer operation tests.
+
+* copy_with_various_rows_per_image_and_bytes_per_row: covers
```

Comment:
This test plan is a bit dry, can we explain with more prose what happens? Often this would mean moving the large comment that's before the tests here, and explains the special cases in a bit more detail.

Reply:
Done